x-pack/plugins/canvas/public/state/store.js

```diff
@@ -12,9 +13,21 @@ import { getRootReducer } from './reducers';
 let store;
 
 export function createStore(initialState) {
-  if (typeof store !== 'undefined') throw new Error('Redux store can only be initialized once');
+  if (typeof store !== 'undefined') {
+    throw new Error(
+      i18n.translate('xpack.canvas.state.reduxStoreCanBeInitializedOnceErrorMessage', {
```

Comment:
xpack.canvas.state.reduxStoreCanBeInitializedOnceErrorMessage => xpack.canvas.state.reduxStoreMultipleInitializationErrorMessage

I know that there are a lot of cases when we can name id only using text we translate but in some cases like this one we can name id using sense that text has.